lib/rubocop-rspec.rb

```diff
@@ -40,10 +39,13 @@
 require_relative 'rubocop/rspec/factory_bot'
 require_relative 'rubocop/rspec/hook'
 
-RuboCop::RSpec::Inject.defaults!
-
 require_relative 'rubocop/cop/rspec_cops'
 
+project_root = File.join(__dir__, '..')
+RuboCop::ConfigLoader.inject_defaults!(project_root)
+obsoletion = File.join(project_root, 'config', 'obsoletion.yml')
+RuboCop::ConfigObsoletion.files << obsoletion
```

Comment:
We do this already in https://github.com/rubocop/rubocop-rspec/blob/495e175da98b86be2c13fcd15cf3808f707094ca/lib/rubocop/rspec.rb#L11-L12. Is the plan to remove that file entirely?